### PR TITLE
Clarify networking requirements for SecureDrop installs

### PR DESCRIPTION
**Pull request by joaedwar**
Pull requests should be opened against the `main` branch. For more information on contributing to SecureDrop documentation, see the [Documentation Guidelines](https://docs.securedrop.org/en/stable/development/documentation_guidelines.html).

## Status
*(What state is your PR in? Select one of the following and delete the option that does not apply.)*

Ready for review 

## Description of Changes

* Description: Update note for network requirements regarding corporate LAN

* Fixes [Issue#4609](https://github.com/freedomofpress/securedrop/issues/4609) (of securedrop repo)

## Testing
*(How should the reviewer test this PR? Write out any special testing steps here.)*

* n/a

## Release 
*(Any special considerations for release of this change into the stable version of the documentation?)*

* n/a 

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [x] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/development/i18n.rst [hunk 1]
```diff
@@ -359,7 +359,7 @@ Inspect the screenshots in the directory ``securedrop/tests/pageslayout/screensh
 and make sure that their content corresponds to the expected version of the
 codebase.
 
-`Obtain your API key <https://weblate.securedrop.org/accounts/profile/#api>`__
+`Obtain your API key <https://weblate.securedrop.org/accounts/login/?next=/accounts/profile/#api>`__
```
Comment (eloquence) [inline docs/development/i18n.rst:362]: Did you intentionally modify this URL?

Comment (joaedwar) [reply docs/development/i18n.rst:362]: yeah. It was coming up as a fail in the docs link check

Comment (eloquence) [reply docs/development/i18n.rst:362]: Got it, thanks for clarifying. :) The issue here is that this link requires being logged into Weblate. I'd prefer not to rely on Weblate's login redirect logic just to make the link checker happy. You can just add it to the ignore-list for the linkcheck, here, which is generally what we do for links that require authentication: 

https://github.com/freedomofpress/securedrop-docs/blob/main/docs/conf.py#L320-L326

**Comment by eloquence** [comment]
@zenmonkeykstop Would be good to have your eyes on this one :-)

**Review by eloquence — CHANGES_REQUESTED** [review]
(See inline comment re: Weblate link)

File: docs/development/i18n.rst (hunk 1 above)
Comment (joaedwar) [reply docs/development/i18n.rst]: ok. reverted back

**Review by zenmonkeykstop — DISMISSED** [review]
LGTM, it's a good clarification. Ok to go after rebase.

**Review by eloquence — APPROVED** [review]

**Comment by eloquence** [comment]
Thanks @joaedwar!

**Pull request merged**